src/Microsoft.DotNet.Interactive.Jupyter/JupyterInteractiveHost.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Interactive.Jupyter.Protocol;
+
+namespace Microsoft.DotNet.Interactive.Jupyter
+{
+    public class JupyterInteractiveHost : IInteractiveHost
+    {
+        public JupyterInteractiveHost()
+        {
+        }
+
+        public Task AddCellAsync(string language, string contents, CancellationToken cancellationToken = default)
```

Comment:
A good test of the naming here is whether this works conceptually with a REPL as well. 

I think this should likely not be part of the interface but instead be just another command that kernels can choose to implement.

And then maybe there's a generalization that can work across notebooks and REPLs, `DisplayEditableSubmission` or something.

Reply:
This is purely a feature of the front end, and not of the kernel.  Cells can be added in VS Code, but not in Jupyter.  Once @colombod and I have finished the various kernel refactorings, this probably _should_ move to the VS Code Kernel and GetInput should move to the VS Code and Jupyter kernels, but in the mean time we needed a place to dump the methods.

Reply:
In the Jupyter namespace, "cell" is a reasonable word. 

But `IInteractiveHost` (which is in `Microsoft.DotNet.Interactive`) has "cell"-related methods, which I think is the wrong level of abstraction. We should look for something that makes sense in interfaces other than notebooks. The command naming should align to that. It would be better to get the naming right sooner than later as other code will be depending on this soon.